mmaction/models/backbones/resnet3d_slowonly.py

```diff
@@ -35,23 +36,7 @@ def __init__(self,
             conv1_stride_t=conv1_stride_t,
             pool1_stride_t=pool1_stride_t,
             inflate=inflate,
+            with_pool2=with_pool2,
             **kwargs)
 
         assert not self.lateral
-
-    def forward(self, x):
```

Comment:
why removed

Reply:
Found this `forward()` method is redundant when we assign `with_pool2=False`